scss/civicrm/search/pages/_results.scss

```diff
@@ -58,14 +60,26 @@
     margin: 0;
 
     > .element-right {
+      margin-right: 20px !important;
       margin-top: 7px;
     }
+  }
 
-    + .float-right {
-      float: none !important;
-      padding: 4px 10px 10px !important;
-      text-align: right;
-    }
+  .float-right {
+    float: none !important;
+    padding: 10px 20px 20px !important;
+    text-align: right;
+  }
+
+  .ui-spinner-input {
```

Comment:
but this should be done in any page, right? Shouldn't this be in a more generic partial?

Reply:
Yes, that makes sense, I will move